cmake/tfc_cpack.cmake

```diff
@@ -26,22 +22,12 @@ set(CPACK_PACKAGE_VERSION_PATCH ${PROJECT_VERSION_PATCH})
 set(CPACK_VERBATIM_VARIABLES YES)
 
 #set(CMAKE_INSTALL_PREFIX "/usr" CACHE PATH "Installation prefix" FORCE)
-set(CPACK_PACKAGING_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX})
+#set(CPACK_PACKAGING_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX})
 
 if(CMAKE_STRIP)
   set(CPACK_RPM_SPEC_MORE_DEFINE "%define __strip ${CMAKE_STRIP}")
 endif()
 
-# set dependencies to hosts libc version and cockpit for UI
-if (LDD_RESULT EQUAL 0)
```

Comment:
I would like to fix this instead of removing, it is not fun to debug missing shared libraries while running tfc executables. It is more straight forward to get the error while installing the package

Reply:
libc6 is not specific enough we need to then have the version also, cockpit is not a missing shared library, you can run all of tfc perfectly fine without it.

Reply:
Yes remove cockpit and use libc6 and even better add libstdc++ with their according version. For now maybe only remove cockpit?

Reply:
libc6 added back with version